swarm/network/stream/stream.go

```diff
@@ -388,14 +388,6 @@ func (r *Registry) Quit(peerId enode.ID, s Stream) error {
 	return peer.Send(context.TODO(), msg)
 }
 
-func (r *Registry) NodeInfo() interface{} {
-	return nil
-}
-
-func (r *Registry) PeerInfo(id enode.ID) interface{} {
```

Comment:
The above comment applies here as well, this is an optional protocol information function; so it should be handled the same way as whatever we decide with `NodeInfo`

Reply:
Yes, this is the same dead code as the one above.

Reply:
can safely remove